### PR TITLE
feat: add Swagger dependencies, endpoint, example JSDoc

### PR DESCRIPTION
**Pull request by parkedwards**
Also:
-- fixes `docker-compose.yaml` for postgres
-- updates tsconfig to allow JSON import

Resolves witsawa-corporation/node-ts-boilerplate#6

image

### CODE REVIEW

**Comment by parkedwards** [comment]
@ak1103dev ready for review

File: app/routes/user.ts [hunk 1]
```diff
@@ -5,6 +5,24 @@ import { authentication } from 'roleResolvers'
 
 const router = Router()
 
+/**
```
Comment (ak1103dev) [inline app/routes/user.ts:8]: Why no api for try out

**Review by ak1103dev — COMMENTED** [review]
Please fix the api document to the example image in the following link
https://swagger.io/tools/swagger-ui/

File: app/routes/user.ts (hunk 1 above)
Comment (parkedwards) [reply app/routes/user.ts:8]: @ak1103dev done

**Comment by parkedwards** [comment]
@ak1103dev updated. please review

**Pull request merged**